.travis.yml

```diff
@@ -7,3 +7,13 @@ install:
 script:
     - flake8 kcidb
     - pylint kcidb
+    - find -name '*.yaml' | xargs yamllint
+    - kcidb-tests-validate --urls < tests.yaml
+    - |
+        sort_diff=$(diff -u <(kcidb-tests-list < tests.yaml) \
```

Comment:
That works until `kcidb-tests-list` gets "improved" to start printing the list in alphabetical order ;)

Reply:
True, but it won't need to be improved if we keep this in :grin:
And the point is to have the YAML sorted, not the tool's output.
We actually might not need the tool if we get rid of categories.
The YAML file will be clear enough by itself.